applications/sandbox/sandbox.vcxproj

```diff
@@ -152,7 +152,7 @@
   </ItemDefinitionGroup>
   <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='Release-VK|x64'">
     <ClCompile>
-      <WarningLevel>EnableAllWarnings</WarningLevel>
+      <WarningLevel>Level3</WarningLevel>
```

Comment:
My fault, I meant to change them on every target (it removes the clang C++98 warnings), will fix

Reply:
The extra weird warnings are also automatically ignored when `core/platform.hpp` (or `core/core.hpp`) is included. It would also give you access to all other utility macros such as platform and compiler detection and `L_NODISCARD`, `L_FALLTHROUGH`, `L_MAYBEUNUSED`, `L_ALWAYS_INLINE`, `RULE_OF_5`, `NDOXY`, `L_WARNING`, `L_ERROR` (and ofc in the case of `core/core.hpp` the rest of the legion core engine.